charts/grafana/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: grafana
-version: 6.32.18
+version: 6.33.0
 appVersion: 9.0.5
```

Comment:
Maybe app version should be bumped to 9.1.1, since Unified provisioning is supported only from Grafana 9.1.0?

Reply:
Even though I use it too with 9.1, i did not feel I am qualified to push the major version within this PR. I assume further compatibility testing is done by the maintainers for such upgrade.

For users this can simply be overwritten by defining a custom values.yaml with:
```yaml
image:
  tags: "9.1.1"
  ```